src/BlackLinter.php

```diff
@@ -53,42 +52,23 @@ public function getVersion() {
     return trim(str_replace('black, version', '', $stdout));
   }
 
-  public function getLinterConfigurationOptions() {
-    $options = array(
-      'black.linelen' => array(
-        'type' => 'optional int',
-        'help' => pht('Specify a non-default line length for black'),
-      ),
-    );
-    return $options + parent::getLinterConfigurationOptions();
-  }
-
-  public function setLinterConfigurationValue($key, $value) {
-    switch ($key) {
-      case 'black.linelen':
-        $this->linelen = $value;
-        return;
-    }
-    return parent::setLinterConfigurationValue($key, $value);
-  }
-
-  protected function getMandatoryFlags() {
-    $flags = array('--quiet');
-    if ($this->linelen != null) {
-        array_push($flags, '--line-length', $this->linelen);
-    }
-    return $flags;
-  }
-
   public function getInstallInstructions() {
     return pht('pip3 install black');
   }
 
   protected function parseLinterOutput($path, $err, $stdout, $stderr) {
-    if ($err == 123 or $stderr) {
```

Comment:
I think we still want to considered `$err` and/or `$stderr` so that we don't emit a lint message for files that are already correctly formatted.

Reply:
Added back in